job/server/src/main/java/alluxio/job/migrate/MigrateDefinition.java

```diff
@@ -246,7 +246,7 @@ private static String computeTargetPath(String path, String source, String desti
   private void migrateDirectory(String path, String source, String destination) throws Exception {
     String newDir = computeTargetPath(path, source, destination);
     mFileSystem.createDirectory(new AlluxioURI(newDir),
-        CreateDirectoryPOptions.getDefaultInstance());
+        FileSystemOptions.createDirectoryDefaults(ServerConfiguration.global()));
```

Comment:
Doesn't mFileSystem do the merging already?

Reply:
fixed